### ISSUE TITLE
FilelistsOutput test

### ISSUE THREAD

**Issue by olenagerasimova** [artipie/rpm-adapter#321]
Create test for `FilelistsOutput`, check `PrimaryOutputTest` as an example.

**Comment by genryxy** [comment]
@olenagerasimova I change part of class `PrimaryOutputTest `, but get such error:
`Expected namespace uri 'http://linux.duke.edu/metadata/filelists' but was 'null'`

In `PrimaryOutputTest` `.xml` file has two namespaces therefore the tests are passed. Should I ignore default namespace by this way:
` .withNodeFilter(
                                node -> !node.isDefaultNamespace(node.getNamespaceURI()) 
                        )`

**Comment by olenagerasimova** [comment]
@genryxy 
> I change part of class PrimaryOutputTest

why have you done this? There is not need to change `PrimaryOutputTest`, it's necessary to create test for `FilelistsOutput`.

Please, add headers to `abc-filelists.xml.example` and `libdeflt-filelists.xml.example` with `filelists` tag and `namespace`:
```
<?xml version="1.0" encoding="UTF-8"?>
<filelists xmlns="http://linux.duke.edu/metadata/filelists" packages="1">
<---file content--->
</filelists>
```

As for other differences between generated xml and examples, please, filter/ignore them with the help of `withNodeFilter` and `withAttributeFilter` methods. One of these tests' goal is to figure out what is missing/wrong in our xml versions and fix it later.

**Comment by genryxy** [comment]
@olenagerasimova 
Yes, sorry for misunderstanding. I created a new class `FilelistsOutput` and in this class write code. 
Thanks

**Comment by olenagerasimova** [comment]
closed via #337

**Comment by 0crat** [comment]
Job was finished in 22 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))
